### DISCUSSION TITLE
github_trending_2023-12-20/

### DISCUSSION THREAD

**Discussion by giscus** [hbstack/theme#668] [General]
# github_trending_2023-12-20/

微博热搜榜 排名 1.习近平对甘肃6.2级地震作出重要指示 排名 2.甘肃地震 最糟糕时间最薄弱地方 排名 3.网友吐槽买了一盒金车厘子583元 排名 4.甘肃加油青海加油 排名 5.奇瑞高端新能源星纪元ES全球上市 排名 6.特朗普被裁决不具备总统选举资格 排名 7.泰国瑞幸向中国瑞幸索赔100亿泰铢 排名 8.地震

https://www.jobcher.com/github_trending_2023-12-20/

**Comment by jobcher** [comment]


**Reply by razonyang** [reply]
@jobcher 温馨提醒：你需要将配置中 `giscus` 相关的参数修改为你的，这样他人评论时，你才会收到通知（GitHub 通知），相关文档：

- https://hbstack.dev/modules/blog/comments/giscus/
- https://hbstack.dev/modules/docs/comments/giscus/
- https://hugomods.com/blog/2023/05/how-to-configure-giscus/。
